src/test/rustdoc/spotlight-from-dependency.rs

```diff
@@ -0,0 +1,24 @@
+#![crate_name = "foo"]
+
+use std::iter::Iterator;
+
+// @has foo/struct.Odd.html
+// @has - '//span[@class="notable-traits"]//code/span' 'impl Iterator for Odd'
```

Comment:
nit: Might be clearer if this `@has` check were next to `pub fn new()` instead, to make it clear that the dialog is for the function.

Reply:
Added it.

Reply:
Ah, I meant move the physical `@has` comment. But what you did is fine too :)